### ISSUE TITLE
missing PubSubClient.h ?

### ISSUE THREAD

**Issue by pcopa** [SuperHouse/BasicOTARelay#2]
Thanks for your Blogs.
Just getting started with Sonoff hacks and have gotten this far.
It looks like the #include <PubSubClient.h> can not be found while compiling  in IDE.
What do I need to load?

**Comment by SabreFrenzy** [comment]
HI,

Have just started playing with this code also.  You can add the PubSubClient.h in the IDE:

Sketch>Include Library>Manage Library>
Then search for PubSubClient.h in the list select it and install it.

Hope that helps

**Comment by pcopa** [comment]
Thanks @SabreFrenzy.  Supper, that did it. 
Just getting started with pi, openhabian and MQTT

**Comment by SabreFrenzy** [comment]
I've got the point where I have flashed my sonoff with the code and it's on my network but can't get it to subscribe to my MQTT.  Have you managed it?

**Comment by pcopa** [comment]
@SabreFrenzy  Have put MQTT on the side for now, because flashing the sonoff did not work.
I've spent 2 days trying to find a stable way to flash the sonoff.  Have tried every thing from new cables to adding power supplies, but nothing worked. 
Only after I had to do a reboot, for another reason this afternoon, it worked fine (ones). I've found out that I can only  program the sonoff ones and only after a reboot of my WIN 10 and Ardino 1.8.3 system. After that only one led blinks 6 times on the USB to serial device and then "no sync" error pops up.   
Will pick up MQTT tomorrow.
